src/organisation/store/actions/organisation.actions.ts

```diff
@@ -5,6 +5,9 @@ export const LOAD_ORGANISATION = '[Organisation] Load Organisation';
 export const LOAD_ORGANISATION_SUCCESS = '[Organisation] Load Organisation Success';
 export const LOAD_ORGANISATION_FAIL = '[Organisation] Load Organisation Fail';
 
+export const UPDATE_ORGANISATION_PBA_PENDING_ADD = '[Organisation] Update Pending PBA addition to organisation';
+export const UPDATE_ORGANISATION_PBA_PENDING_REMOVE = '[Organisation] Update Pending PBA removal to organisation';
```

Comment:
Should it be "removal _from_ organisation" rather than "removal to organisation"?